spec/bagger_tag_spec.rb

```diff
@@ -0,0 +1,137 @@
+require "spec_helper"
+require "bagger_tag"
+
+describe BaggerTag do
+
+
+  describe "::from_hash" do
+    let(:json_tag) do
+      JSON.parse(<<JSON)
+      {
+        "SomeField": {
+          "fieldRequired": true,
+          "valueList": [
+            "allowed_value",
+            "another_allowed_value"
+          ]
+        }
+      }
+JSON
+
+  end
```

Comment:
Some indentation issues here. Also, I've never seen the (<<XYZ) format -- neat. A little odd to have it match the class name, though.

Reply:
The rubocop style guide requires using meaningful names for the heredoc delimiters. This also enables proper syntax highlighting with editor support.

I'll fix the indentation issues.